README.md

```diff
@@ -121,7 +121,7 @@ Clone this repository to your desired folder:
 
```

Comment:
-  As per the project requirements, in your README file, you need to mention the API that your application is consuming. In your case, your application is consuming the Leaderboard API provided to you by Microverse. Kindly add the link of the API in your README file, mentioning that your application is using this particular API to store and fetch data.
![image](https://user-images.githubusercontent.com/33268927/224482942-c7f882db-744b-49df-8579-19b8002a939c.png)